.rubocop_todo.yml

```diff
@@ -80,10 +81,12 @@ Metrics/LineLength:
 Metrics/MethodLength:
   Max: 56
 
-# Offense count: 1
+# Offense count: 2
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
   Max: 472
```

Comment:
@Ana06 but then we just propagate the same problem as https://github.com/openSUSE/osem/issues/1470#issuecomment-298690605